### PR TITLE
Show tooltip containing widget description when hovering over a widget

### PR DESCRIPTION
**Pull request by prasadwalvekar**
### Changes

- Wrapped JSX for the widget component inside `<OverlayTrigger />` component to show tooltip when user hovers over a widget.

### Screenshot

Screenshot 2021-06-25 at 6 31 43 PM

### CODE REVIEW

**Comment by prasadwalvekar** [comment]
@Navaneeth-pk Please review.

File: frontend/src/Editor/Box.jsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ import { Multiselect } from './Components/Multiselect';
 import { Modal } from './Components/Modal';
 import { Chart } from './Components/Chart';
 import { Map } from './Components/Map';
+import OverlayTrigger from 'react-bootstrap/esm/OverlayTrigger';
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Box.jsx:18]: Just curious, what is the difference between this line and `import OverlayTrigger from 'react-bootstrap/OverlayTrigger';`?

Comment (prasadwalvekar) [reply frontend/src/Editor/Box.jsx:18]: Just noticing this as it was auto-imported by VS Code. Both are the same, the only difference being one is a CJS module (Common JS) and the other is an EcmaScript module (ESM). 
The react-bootstrap library is using multiple formats for distribution.

Comment (prasadwalvekar) [reply frontend/src/Editor/Box.jsx:18]: Refactoring the import now to keep it consistent with other places where <OverlayTrigger /> has been imported.

**Pull request merged**